### PR TITLE
Setup environments

### PR DESCRIPTION
**Pull request by nbowes24**
These changes include getting the nickname from the claim but primarily consist of changes for the Cosmos DB Options.

### CODE REVIEW

File: src/SFA.DAS.SelfService.Infrastructure/Client/ReleaseClient.cs [hunk 1]
```diff
@@ -127,10 +142,23 @@ public static VstsRelease CreateRelease(ReleaseHttpClient releaseClient, int rel
...
+                ReleaseDefininitionId = release.ReleaseDefinitionReference.Id,
+                ReleaseEnvironments = release.Environments.Select(x => new VstsEnvironment { DefinitionId = x.DefinitionEnvironmentId, EnvironmentReleaseId = x.Id, Name = x.Name }).ToList()
             };
 
             return vstsRelease;
         }
+
+        public void StartEnvironmentDeployment(VstsRelease vstsRelease, int releaseEnvironmentId)
+        {
+            ReleaseHttpClient releaseClient = _vssConnection.GetClient<ReleaseHttpClient>();
```
Comment (adam230594) [inline src/SFA.DAS.SelfService.Infrastructure/Client/ReleaseClient.cs:154]: is this supposed to be vss or vsts?

Comment (nbowes24) [reply src/SFA.DAS.SelfService.Infrastructure/Client/ReleaseClient.cs:154]: Good spot, check out line 18
private readonly VssConnection _vssConnection;

The object from the package is VssConnection :( It stands for VisualStudio.Services
using Microsoft.VisualStudio.Services.ReleaseManagement.WebApi;

File: README.md [hunk 2]
```diff
@@ -1,10 +1,4 @@
 # das-self-service
 
-|             |                        |
-|-------------|------------------------|
-| Build       ||
-| Development ||
-| Production  ||
```
Comment (EwanNoble) [inline README.md:7]: where badges

File: src/SFA.DAS.SelfService.Infrastructure/Client/ReleaseClient.cs [hunk 3]
```diff
@@ -65,15 +65,30 @@ public VstsReleaseDefinition GetRelease(string releaseName)
             return vstsRelease;
         }
 
-        public VstsReleaseStatus CheckReleaseStatus(int releaseDefinitionId, int releaseId)
+        public VstsReleaseDefinition GetRelease(int releaseDefinitionId)
+        {
+            var client = _vssConnection.GetClient<ReleaseHttpClient>();
+
+            var release = client.GetReleaseDefinitionAsync(_configuration.Value.ProjectName, releaseDefinitionId).Result;
```
Comment (EwanNoble) [inline src/SFA.DAS.SelfService.Infrastructure/Client/ReleaseClient.cs:72]: Can we do async properly instead?

File: README.md (hunk 2 above)
Comment (nbowes24) [reply README.md:7]: Badges added 👍

**Pull request merged**